components/UI/Map/LeafletMap.tsx

```diff
@@ -68,18 +73,21 @@ const MapEvents = () => {
           EXPAND_COORDINATE_BY_VALUE
         );
       }
-
       setCoordinates(localCoordinates, eventType);
+      const _lat = localCoordinates.getCenter().lat;
+      const _lng = localCoordinates.getCenter().lng;
+      const _zoomLevel = zoomLevel;
 
       // set cordinates and zoom level to url
-      const cordinatesURL = `#lat=${localCoordinates.getCenter().lat}&lng=${
-        localCoordinates.getCenter().lng
-      }&zoom=${zoomLevel}`;
-
-      window.history.replaceState(
-        { ...window.history.state, as: cordinatesURL, url: cordinatesURL },
-        "",
-        cordinatesURL
+      const cordinatesURL = `lat=${_lat}&lng=${_lng}&zoom=${_zoomLevel}`;
```

Comment:
`new URLSearchParams({ lat, lang, zoom }).toString()` olarak oluşturursak ileride developerlarin yapabilecegi hataları engellemiş oluruz.

Reply:
Ve variable'in ismi de yanlis. Coordinates URL degil, coordinates ve zoom ile olusturdugumuz search query.